### PR TITLE
Create B827EBFFFE9CA0EF.json

### PR DESCRIPTION
**Pull request by sanjeevadesilva**
lora

### CODE REVIEW

File: B827EBFFFE9CA0EF.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "gateway_ID": "B827EBFFFE9CA0EF",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 0,
```
Comment (gonzalocasas) [inline B827EBFFFE9CA0EF.json]: Please use the real coordinates of the gateway's placement.

File: B827EBFFFE9CA0EF.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "ref_longitude": 0,
+    "ref_altitude": 0,
+    "contact_email": "shsilva@gmail.com",
+    "description": "Here a human readable description of it"
```
Comment (gonzalocasas) [inline B827EBFFFE9CA0EF.json]: And if possible, update the placeholder description ;)

**Comment by sanjeevadesilva** [comment]
done

**Comment by gonzalocasas** [comment]
Duplicate

**Pull request closed without merging**